src/loaders/google_map_loader.js

```diff
@@ -16,6 +15,11 @@ export default (bootstrapURLKeys, heatmapLibrary) => {
     return _customPromise;
   }
 
+  // avoid api to be loaded multiple times
+  if (loadPromise_) {
```

Comment:
@jpoehnelt Instead of checking for `!loader_` we return the promise, avoiding the loader to add another script